src/main/java/org/folio/consortia/service/impl/SyncPrimaryAffiliationServiceImpl.java

```diff
@@ -46,7 +46,7 @@ public void syncPrimaryAffiliations(UUID consortiumId, String tenantId, String c
     log.info("Start syncing user primary affiliations for tenant {}", tenantId);
     List<User> users = new ArrayList<>();
     try {
-      users = userService.getUsersByQuery("cql.allRecords=1", 0, Integer.MAX_VALUE);
+      users = userService.getUsersByQuery("(cql.allRecords=1 NOT type =\"patron\")", 0, Integer.MAX_VALUE);
```

Comment:
Checked in Volaris team, dcb users should not be processed by consortia pipeline also, related PR: https://github.com/folio-org/mod-users/pull/311/files#diff-c61ac265e749725833c34e88681dd23fd2c267c2de0c62217a27526f50395772R9 

Basically it will be new users  with this type, and all existing users will be without it, but if imagine situatuion that consortia deployed for some customer for example for Q and dcb - for Poppy, and in Q customer decides to enable ECS , so to be on the safe side better to exclude dcb users as well

Reply:
Looking..